applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -46,7 +52,10 @@ def __init__(self, Model, settings):
             for node in cond.GetNodes():
                 node.Set(KratosMultiphysics.SOLID)
 
-    def ExecuteInitialize(self):
+    def SolveSolutionStep(self):
```

Comment:
The function solvesolutionstep does not exist in the base class. I understand you want to make the naming consisten because you are calling this function within the SolveSolutionStep of your solver, however:

-  Please use one of the functions in the base class: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/python_process.py
- Please check that this change doesnt break the standard potential solver (have you check that the tests in the CompressiblePotentialFlowApplication?)

Reply:
I set it as ExecuteBeforeSolutionLoop(self)
do you think it is fine?